README.md

```diff
@@ -136,7 +136,8 @@ class CloneOptions {
 }
 
 int Main(string[] args) {
-  return CommandLine.Parser.Default.ParseArguments<AddOptions, CommitOptions, CloneOptions>(args)
```

Comment:
I suggest leaving this as is, could we limit this PR to adding the F# example? Feel free to force push if you want clean history.